docs/sources/project/review-pr.md

```diff
@@ -49,9 +49,12 @@ need to update your pull request with additional changes.
 
 To update your existing pull request:
 
-1. Change one or more files in your local `docker-fork` repository.
+1. Change one or more files in your local `docker-fork` repository
```

Comment:
Thank you for the good catch.  The text has some issues.  Parentheticals as used here can indicate problems in  structure particularly in designing step text. These are being used to include system results and extra steps.  They need to extracted out, clarified, and positioned correctly.

--> Markdown: https://gist.github.com/moxiegirl/7bde985ae88f91ce36a2

To update your existing pull request:

1. Checkout the PR branch in your local `docker-fork` repository.

  This is the branch associated with your request.
	
2. Change a file and then stage it.

  The command syntax is:
 
       git add <filename>

3. Commit the change with the `git commit --amend` command.

    	$ git commit --amend 

    Git opens an editor containing your last commit message.

3. Adjust your last comment to reflect this new change.

	Here is an example:

        Added a new sentence per Anaud's suggestion	

        Signed-off-by: Mary Anthony <mary@docker.com>

        # Please enter the commit message for your changes. Lines starting
        # with '#' will be ignored, and an empty message aborts the commit.
        # On branch 11038-fix-rhel-link
        # Your branch is up-to-date with 'origin/11038-fix-rhel-link'.
        #
        # Changes to be committed:
        #		modified:   docs/sources/installation/mac.md
        #		modified:   docs/sources/installation/rhel.md

4. Force push the change to your branch.
   
   The command syntax is:

        git push -f origin <branch_name>